CHANGES.md

```diff
@@ -1,3 +1,8 @@
+# Changes in Version 3
+
+## Summary of breaking changes
+- `omero_server_system_umask` defaults to `0022`. To restore the old behaviour set it to `0002`.
```

Comment:
Do we do American spelling for our playbook comments?

Reply:
What the policy for code comments in the rest of our repos?

Reply:
@hflynn: How picky are we about docs like https://github.com/openmicroscopy/ansible-role-omero-server/blob/2.0.3/CHANGES.md?

Reply:
For this repo, I don't personally mind, but our official spelling is US.